.eslintrc.js

```diff
@@ -83,5 +86,18 @@ module.exports = {
         browser: true,
       },
     },
+    {
+      files: [
+        '**/forma-36-website/**/*',
+        '**/forma-36-react-components/tools/**/*',
+        '**/forma-36-react-components/**/Icon/svg/*',
+        '**/forma-36-react-components/**/ProductIcon/**/*',
+        '**/forma-36-react-components/**/Illustration/**/*',
+        '**/packages/components/**/*',
```

Comment:
🤔 why not just the `./src/**`?

Reply:
That would exclude all the components we want to lint.